draft-looker-oauth-attestation-based-client-authentication.md

```diff
@@ -299,6 +299,41 @@ This section registers the value "attest_jwt_client_auth" in the IANA "OAuth Tok
 
 --- back
 
+# Additional Examples
+
+## Wallet Instance Attestation
+
+This non-normative example shows a client attestations used as an wallet instance attestation in the context of eIDAS 2.0, e.g. to secure a Type-1 configuration credential. The additional claims describe the wallet's device binding und user binding capabilities and the achievable level of assurance.
+
+~~~
+{
+	"typ": "wallet-attestation+jwt",
+	"alg": "ES256",
+	"kid": "1"
+}
+.
+{
+	"iss": "https://attestation-service.ssi.tir.budru.de",
+	"sub" : "https://lissi.org",
+	"iat": 1541493724,
+	"exp": 1516247022,
+	"wallet_name": "Lissi Dev",
+	"wallet_version": "1.6.0",
```

Comment:
```suggestion
```

Why the attestation should disclose information like these?
Probably the User doesn't want to disclose the device name and the wallet version

Reply:
"wallet_name" is meant as a human readable name of the wallet solution, the technical identifier for a wallet solution would be registered as "sub"
"wallet_version" is the version of the wallet instance of the wallet solution. The certification and validity status of a wallet solution probably depends on the version, thus for lookup in a trust registry the version is useful

Reply:
I see some value in the wallet name. wrt the version, as we discussed yesterday, I would assume whatever is needed to lookup data about the authorization of the wallet should be encoded in the iss or sub value.

Reply:
added version to sub